profile/src/hierarchical_deterministic/bip32/hd_path.rs

```diff
@@ -0,0 +1,229 @@
+use crate::prelude::*;
+
+use slip10::path::BIP32Path;
+
+#[derive(
+    Clone,
+    PartialEq,
+    Eq,
+    Hash,
+    PartialOrd,
+    Ord,
+    SerializeDisplay,
+    DeserializeFromStr,
+    derive_more::Display,
+    derive_more::Debug,
+    uniffi::Record,
+)]
+#[display("{}", self.to_bip32_string())]
```

Comment:
[derive_more](https://jeltef.github.io/derive_more/derive_more/display.html) crate, nice to get more derives, but unfortunately seems not be possible to use `#[displayAndDebug(..)]` why we need to lines.